### DISCUSSION TITLE
New repository for every project

### DISCUSSION THREAD

**Discussion by lologog** [community/community#22474] [New to GitHub]
<p>Do I have to create a new repository for every project? For example if I have like 40 projects that I want to put on github then I have to create a new repository per project? Maybe it is possible to create something like folders/catalogs?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Yes, separate repositories for separate projects is strongly recommended. Putting multiple projects into one repository tends to cause <em>a lot</em> of trouble. :sweat_smile:</p>
<p>And no, GitHub doesn’t support a folder structure for repositories.</p>
